aws/resource_aws_cloudtrail.go

```diff
@@ -357,3 +422,93 @@ func cloudTrailSetLogging(conn *cloudtrail.CloudTrail, enabled bool, id string)
 
 	return nil
 }
+
+func cloudTrailSetEventSelectors(conn *cloudtrail.CloudTrail, d *schema.ResourceData) error {
+	input := &cloudtrail.PutEventSelectorsInput{
+		TrailName: aws.String(d.Id()),
+	}
+
+	eventSelectors := expandAwsCloudTrailEventSelector(d.Get("event_selector").([]interface{}))
+	input.SetEventSelectors(eventSelectors)
```

Comment:
Nitpick: We generally do not use the `TYPE.SetXXX` (setter) functions of the SDK across this repository -- you can use `input.EventSelectors = eventSelectors` instead 😄